types/react-native/index.d.ts

```diff
@@ -8688,12 +8688,12 @@ export namespace Appearance {
     /**
      * Add an event handler that is fired when appearance preferences change.
      */
-    export function addChangeListener(listener: AppearanceListener): EventSubscription;
+    export function addChangeListener(listener: AppearanceListener): void;
```

Comment:
Could you add tests to cover those changes?

Reply:
I wasn't able to run the tests locally so I didn't invest more time on it. I'd rather not spend much more time on this.

`Error: Cannot find module '/Users/brunolemos/.dts/typescript-installs/3.3/node_modules/typescript'`

Reply:
I think we could skip a change to the test here: `Appearance.addChangeListener` and `Appearance.addChangeListener` is both called and there are no assumptions made that this was returning a `EventSubscription` (e.g. by calling `remove()`). I would therefore argue that it’s fine to skip a test for checking it returns `undefined`. What do you think @a-tarasyuk ?